cmd/cleantool/cleantool.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/v52/github"
+	"github.com/sethvargo/go-githubactions"
+	"golang.org/x/oauth2"
+)
+
+func main() {
+	ctx := context.Background()
+	tokenName := "ROBOT_TOKEN"
+	token := os.Getenv(tokenName)
+	if token == "" {
+		log.Fatalf("env variable %v is not found, please set it before run it", tokenName)
+	}
+	ts := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: token},
+	)
+	tc := oauth2.NewClient(ctx, ts)
+	client := github.NewClient(tc)
+	packageType := "container"
+	packageName := "ferretdb-dev"
+	orgName := "FerretDB"
+	daysBack := 90
+	pageSize := 100
+	var versions []string
+	for i := 1; i < 100; i++ {
```

Comment:
Any reason why it wouldn't iterate beyond 100 pages?

If you would like to iterate without an arbitrary limit, you could use `for {` instead, then increment `i++` at the end of the loop. :smile: